qiskit/visualization/pulse/matplotlib.py

```diff
@@ -348,16 +348,17 @@ def _build_channels(self, schedule, channels, t0, tf, show_framechange_channels=
                     snapshot_channels[channel].add_instruction(start_time, instruction)
         return channels, output_channels, snapshot_channels
 
-    def _count_valid_waveforms(self, output_channels, scale=1, channels=None,
+    def _count_valid_waveforms(self, output_channels, scale, channels=None,
```

Comment:
this `scale` is a dict from channels to floats, now?

Reply:
it's back to float or None. if None, auto scaling is applied.